src/inc/quic_platform.h

```diff
@@ -66,80 +66,83 @@ typedef struct CXPLAT_SLIST_ENTRY {
+#define QUIC_POOL_PLATFORM_GENERIC              'A2cQ' // Qc2A - QUIC Platform generic
+#define QUIC_POOL_DATAPATH                      'B2cQ' // Qc2B - QUIC Platform datapath
+#define QUIC_POOL_SOCKET                        'C2cQ' // Qc2C - QUIC Platform socket
+#define QUIC_POOL_STORAGE                       'D2cQ' // Qc2D - QUIC Platform storage
+#define QUIC_POOL_HASHTABLE                     'E2cQ' // Qc2E - QUIC Platform hashtable
+#define QUIC_POOL_HASHTABLE_MEMBER              'F2cQ' // Qc2F - QUIC Platform hashtable member lists
+#define QUIC_POOL_LOOKUP_HASHTABLE              '03cQ' // Qc30 - QUIC Lookup Hash Table
+#define QUIC_POOL_REMOTE_HASH                   '13cQ' // Qc31 - QUIC Remote Hash Entry
+#define QUIC_POOL_SERVERNAME                    '23cQ' // Qc32 - QUIC Server Name
+#define QUIC_POOL_APP_RESUMPTION_DATA           '33cQ' // Qc33 - QUIC App Resumption Data
+#define QUIC_POOL_INITIAL_TOKEN                 '43cQ' // Qc34 - QUIC Initial Token
+#define QUIC_POOL_CLOSE_REASON                  '53cQ' // Qc35 - QUIC Close Reason
+#define QUIC_POOL_SERVER_CRYPTO_TICKET          '63cQ' // Qc36 - QUIC Crypto Server Ticket Buffer
+#define QUIC_POOL_CLIENT_CRYPTO_TICKET          '73cQ' // Qc37 - QUIC Crypto Client Ticket Buffer
+#define QUIC_POOL_CRYPTO_RESUMPTION_TICKET      '83cQ' // Qc38 - QUIC Crypto Resumption Ticket
+#define QUIC_POOL_TLS_BUFFER                    '93cQ' // Qc39 - QUIC Tls Buffer
+#define QUIC_POOL_SEND_REQUEST                  'A3cQ' // Qc3A - QUIC Send Request
+#define QUIC_POOL_API_CTX                       'B3cQ' // Qc3B - QUIC API Context
+#define QUIC_POOL_STATELESS_CTX                 'C3cQ' // Qc3C - QUIC Stateless Context
+#define QUIC_POOL_OPER                          'D3cQ' // Qc3D - QUIC Operation
+#define QUIC_POOL_EVENT                         'E3cQ' // Qc3E - QUIC Event
+#define QUIC_POOL_TLS_PFX                       'F3cQ' // Qc3F - QUIC Platform PFX
+#define QUIC_POOL_DESIRED_VER_LIST              '04cQ' // Qc40 - QUIC App-supplied desired versions list
+#define QUIC_POOL_DEFAULT_COMPAT_VER_LIST       '14cQ' // Qc41 - QUIC Default compatible versions list
+#define QUIC_POOL_VERSION_INFO                  '24cQ' // Qc42 - QUIC Version info
+#define QUIC_POOL_PROCESS                       '34cQ' // Qc43 - QUIC Process
+#define QUIC_POOL_TLS_TMP_TP                    '44cQ' // Qc44 - QUIC Platform TLS Temporary TP storage
+#define QUIC_POOL_PCP                           '54cQ' // Qc45 - QUIC PCP
+#define QUIC_POOL_DATAPATH_ADDRESSES            '64cQ' // Qc46 - QUIC Datapath Addresses
+#define QUIC_POOL_TLS_TICKET_KEY                '74cQ' // Qc47 - QUIC Platform TLS ticket key
+#define QUIC_POOL_TLS_CIPHER_SUITE_STRING       '84cQ' // Qc48 - QUIC TLS cipher suite string
+#define QUIC_POOL_ROUTE_RESOLUTION_WORKER       '94cQ' // Qc49 - QUIC route resolution worker
+#define QUIC_POOL_ROUTE_RESOLUTION_OPERATION    'A4cQ' // Qc4A - QUIC route resolution operation
```

Comment:
I would have shortened `_OPERATION` to `_OPER` so I didn't have to increase the spacing.